### DISCUSSION TITLE
Dropdown Persistence

### DISCUSSION THREAD

**Discussion by ocean1918** [InterStella0/starlight-dpy#10] [Q&A] [answered]
Is there a way to make dropdown persistent, instead of it disappearing when selecting a category?  Found that it defaults to none before pagination.

**Comment by InterStella0** [comment, marked as answer]
EDITED

They disappear because it is a completely new view created and displayed. This is a paginator that is shared with other interfaces. So it is not supported by the library to make the Select object persistent. You can play around with [HelpMenuProvider.provide_cog_view](https://starlight-dpy.readthedocs.io/en/latest/help-command.html#starlight.HelpMenuProvider.provide_cog_view) which provides the paginator for this exact action. It also provides for a cog help so changing this view will affect the cog help interface.

There is also an example on using the help menu provider to modify a help command paginator [here](https://github.com/InterStella0/starlight-dpy/blob/main/examples/live-bot/utils/help_command.py).

**Reply by ocean1918** [reply]
When you chose a category from the dropdown, the dropdown disappears to show cog help embed with pagination buttons and you have to press home for dropdown to appear again. I would want an option to make dropdown persistent, maybe show it instead of home button. Kinda the way dannys bot shows it, if it is possible.
